### PR TITLE
Redirect all logging through slflog4j to logback

### PR DESCRIPTION
**Pull request by nicgrayson**
This redirects all logging to log4j and cjl to slf4j and adds logback classic as the log output. There should probably be a config to turn off the jetty debug logs.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,9 @@
 
     <java.abi>1.7</java.abi>
     <mesos.version>0.20.0</mesos.version>
+    <log4j-over-slf4j.version>1.7.10</log4j-over-slf4j.version>
+    <jcl-over-slf4j.version>1.7.10</jcl-over-slf4j.version>
```
Comment (BenWhitehead) [inline pom.xml]: `slf4j` components are released together with the same versions.  It would be cleaner to have a single `slf4j.version` property.

**Comment by BenWhitehead** [comment]
I can't guarantee that this will be merged, but there are a few notes that I've made.

I'm all for unifying logging to a central backend and configuration. Please define a `logback.xml` in `src/main/resources` since you have set logback as the logging backend.

**Comment by nicgrayson** [comment]
@BenWhitehead I've added a logback.xml and moved over to a single slf4j version var. This just is my preferred way to unify logging.

**Comment by BenWhitehead** [comment]
@nicgrayson Thanks for the updates. 

We're going to test it out here and will let you know if we come up with anything else that needs to be updated.

**Comment by elingg** [comment]
:+1:

**Comment by elingg** [comment]
Piping the logging to logback and using that configuration for all logging is fine by me.

**Pull request merged**